### ISSUE TITLE
Add imagePullSecrets to more ServiceAccounts

### ISSUE THREAD

**Issue by mgladdish** [upmc-enterprises/registry-creds#64]
After much head scratching I think I've worked out that registry-creds only adds `imagePullSecrets` to the default service account in each namespace.

We lock down our pods by specifying our own, more restrictive, service accounts, so we don't get the `awsecr-cred` automatically added and our pods fail to download their image.

Maybe it could be possible to add an annotation to a service account to get it included in registry-creds' sync?

In the meantime, I presume the best/only workaround is to explicitly add the `imagePullSecrets` entry to our own service account definition? It would also be good to make this more explicit in the readme - I've just spent 2 days trying to work out why registry-creds didn't work for one of my pods when it worked for all my others.

**Comment by stevesloka** [comment]
Yup you are correct, an annotation would be an easy path to make this better.
